x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/components/search_bar.tsx

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState, useEffect } from 'react';
+import { IFieldType } from 'src/plugins/data/public';
+// @ts-ignore
+import { EuiSuggest, EuiSuggestItemProps } from '@elastic/eui';
+import { useDebounce } from '../../../hooks';
+import { useStartDeps } from '../../../hooks/use_deps';
+
+const INDEX_NAME = '.kibana';
+
+const DEBOUNCE_SEARCH_MS = 150;
+const HIDDEN_FIELDS = ['agents.actions'];
+
+interface Suggestion {
+  label: string;
+  description: string;
+  value: string;
+  type: {
+    color: string;
+    iconType: string;
+  };
+  start: number;
+  end: number;
+}
+
+interface Props {
+  value: string;
+  fieldPrefix: string;
+  onChange: (newValue: string) => void;
+}
+
+export const SearchBar: React.FunctionComponent<Props> = ({ value, fieldPrefix, onChange }) => {
+  const { suggestions } = useSuggestions(fieldPrefix, value);
+
+  // TODO fix type when correctly typed in EUI
```

Comment:
Can you add comment about or link to a ticket which describes what's missing and how we'll know when we can remove these `any`s and `ignore`s?

Reply:
Thanks @jfsiii for the feedback.
I copied this file from `feature-ingest` branch - under `x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/components/search_bar.tsx`. My goal was to remove this file once `feature-ingest` merged to `master` and use the one in that will come in from that branch (which I assume probably will move to top-level `components` dir).

I can still open up an issue to track sync'ing them up and cleaning it up.  Let me know if I should 😃

Reply:
Oh, I see. Thanks. No need to open a ticket or change the comment. We can deal with this then.

Reply:
cool 👍 . I know @hbharding also has a few suggested changes to the colors being used with the suggestion types